recipe/meta.yaml

```diff
@@ -13,9 +13,9 @@ source:
 
 build:
   skip: True  # [not linux]
-  number: 0
-  # For some reason, compiling with meson, the recommended way, doesn't
-  # correctly allow libva to find libdrm
+  number: 1
+  run_exports:
+    - {{ pin_subpackage('libdrm', max_pin='x.x') }}
```

Comment:
since they are breaking at the 2.4.95 to 2.4.96 level I think the max_pin should go to 'x.x.x'. What do you think?

Patrick

Reply:
Fair enough, but I felt like those looked like "accidental" mistakes that they had exposed an internal API they shouldn't have.

I feel like it "looked" pretty safe. especially over so many releases.